explorer/explorerroutes.go

```diff
@@ -1856,6 +1856,7 @@ func (exp *explorerUI) commonData() *CommonPageData {
 	cd.Version = exp.Version
 	cd.ChainParams = exp.ChainParams
 	cd.BlockTimeUnix = int64(exp.ChainParams.TargetTimePerBlock.Seconds())
+	cd.DevAddress = exp.pageData.HomeInfo.DevAddress
```

Comment:
OK, but note that `exp.pageData` is generally to be protected by a `RWMutex`.  Since `HomeInfo` is a pointer that never changes, and `HomeInfo.DevAddress` never changes either, this is OK.  I might see about changing commonData() and explorerUI so that explorerUI has a CaommonPageData field that gets filled out on explorerUI construction, and only Tip is updated by this function.